### ISSUE TITLE
Small typo in self pace workshop command says python but should say python3

### ISSUE THREAD

**Issue by neoakris** [aws-quickstart/cdk-eks-blueprints#898]
### Describe the documentation issue

I know python2 is EoL at this point, but during the transition it was common for 
pip & python to represent python2
pip3 & python3 to represent python3

I followed the doc as is and on the linked documentation page it mentions pip3 & python, but when copy pasting the command as is (as instructed) it mentions python not found. (I'm guessing an older version of the automatically selected AMI had a symlink to map the cli command `python` to `python3`, but the current latest AMI doesn't make such a distinction and you need to explicitly specify python3.)

So long story short `python` in the doc should be replaced with `python3`.

### Links

https://catalog.workshops.aws/eks-blueprints-for-cdk/en-US/020-setup/self-paced/2-resize-volume

**Comment by shapirov103** [comment]
@youngjeong46 @parkand1 please take a look when you get a chance.

**Comment by youngjeong46** [comment]
@shapirov103 we have moved away from prescriptive instruction and instead are using CloudFormation template to build out the necessary pre-requisites. We can close this issue. @neoakris please re-open if issues persist.
